test/jdk/ProblemList.txt

```diff
@@ -433,7 +433,7 @@ java/awt/xembed/server/RunTestXEmbed.java 7034201 linux-all
 java/awt/Modal/ModalFocusTransferTests/FocusTransferDialogsDocModalTest.java 8164473 linux-all
 java/awt/Frame/DisposeParentGC/DisposeParentGC.java 8079786 macosx-all
 
-java/awt/GraphicsDevice/DisplayModes/CycleDMImage.java 7099223,8274106 macosx-all,linux-all,windows-all
```

Comment:
it seems odd to remove macOS from the list when one of the bugs here is specifically against macOS - https://bugs.openjdk.org/browse/JDK-8274106
If the test still fails on some retina displays, then this needs to remain problem listed.
If not, then remove the reference to 8274106

Wait 7099223 is also about macOS.
So why is this test problem listed on all platforms ?
If it passes always on macOS we also should close out those bugs.
Please check it on all platforms and de-list it for all platforms that pass and if it does fail on Windows or Linux, create a bug for that platform's problems.

Reply:
There are couple of JBS issues related to CycleDMImage.java
[JDK-7099223](https://bugs.openjdk.org/browse/JDK-7099223) - listed for all platforms
[JDK-8312459](https://bugs.openjdk.org/browse/JDK-8312459) - subtask to problemlist on macOS but the JBS number itself is not included in problemlist.
[JDK-8274106](https://bugs.openjdk.org/browse/JDK-8274106) - The one you mentioned earlier.

> Please check it on all platforms and de-list it for all platforms that pass and if it does fail on Windows or Linux, create a bug for that platform's problems.

Checking it again on all platforms.

Reply:
7099223 was noted in 2011 (!) as failing on solaris (doesn't matter anymore) , passing on Linux (so why is linux problem listed ?) and mentions Windows (in 2011) but nothing since then.

Since 2019 it has ONLY been reported on macOS. Perhaps because  it has been problem listed all that time on windows and linux :-), but now is the time to re-check all of this.